### DISCUSSION TITLE
May I use a Stash variable as a filter value?

### DISCUSSION THREAD

**Discussion by chunkitdesign** [Patrick64/reefine#17] [General]
Hi Patrick, thank you for this excellent add-on.
I'm wondering if it's possible to use a Stash variable as a filter value. Specifically, I need to filter classes (in the classes channel) based on whether they have any offerings (from the class offerings channel). Would it be possible?
Thanks,
Chris

**Comment by Patrick64** [comment]
Hi. Thanks, good to hear. Do you want to preselect a filter based on a stash variable? In which case you can use [filter:filter group:default](https://github.com/Patrick64/reefine#filterfilter-groupdefault)
